packages/frontend/pages/vault/[vid].tsx

```diff
@@ -930,34 +937,40 @@ const Component: React.FC = () => {
-                      id="burn-submit-tx-btn"
-                      onClick={() => burn(shortAmountBN)}
-                      className={classes.actionBtn}
-                      size="small"
-                      disabled={action !== VaultAction.BURN_SQUEETH || txLoading || !!adjustAmountError}
-                    >
-                      {action === VaultAction.BURN_SQUEETH && txLoading ? (
-                        <CircularProgress color="primary" size="1rem" />
-                      ) : (
-                        'Burn'
-                      )}
-                    </RemoveButton>
-                    <AddButton
-                      id="mint-submit-tx-btn"
-                      onClick={() => mint(shortAmountBN)}
-                      className={classes.actionBtn}
-                      size="small"
-                      disabled={action !== VaultAction.MINT_SQUEETH || txLoading || !!adjustAmountError}
-                    >
-                      {action === VaultAction.MINT_SQUEETH && txLoading ? (
-                        <CircularProgress color="primary" size="1rem" />
-                      ) : (
-                        'Mint'
-                      )}
-                    </AddButton>
-                  </div>
+                  {!supportedNetwork ? (
+                    <PrimaryButton disabled variant="contained">
+                      Unsupported Network
+                    </PrimaryButton>
+                  ) : (
+                    <div className={classes.managerActions}>
+                      <RemoveButton
+                        id="burn-submit-tx-btn"
+                        onClick={() => burn(shortAmountBN)}
+                        className={classes.actionBtn}
+                        size="small"
+                        disabled={action !== VaultAction.BURN_SQUEETH || txLoading || !!adjustAmountError}
+                      >
+                        {action === VaultAction.BURN_SQUEETH && txLoading ? (
+                          <CircularProgress color="primary" size="1rem" />
+                        ) : (
+                          'Burn'
+                        )}
+                      </RemoveButton>
+                      <AddButton
+                        id="mint-submit-tx-btn"
+                        onClick={() => mint(shortAmountBN)}
+                        className={classes.actionBtn}
+                        size="small"
+                        disabled={action !== VaultAction.MINT_SQUEETH || txLoading || !!adjustAmountError}
+                      >
+                        {action === VaultAction.MINT_SQUEETH && txLoading ? (
+                          <CircularProgress color="primary" size="1rem" />
+                        ) : (
+                          'Mint'
+                        )}
+                      </AddButton>
+                    </div>
+                  )}
                 </div>
               </div>
               <div className={classes.manager}>
```

Comment:
I think we also need to disable lp nft select when user is on unsupported network

Reply:
Fixed this and pushed a commit